lib/player.rb

```diff
@@ -1,16 +1,18 @@
 require 'pry'
 require "./lib/score.rb"
 require "./lib/highest_score.rb"
+require "./lib/tilebag.rb"
 
 module Scrabble
 
   class Player
-    attr_accessor :name, :words_played, :player_score
+    attr_accessor :name, :words_played, :player_score, :tiles
 
     def initialize(name)
       @name = name.capitalize
       @words_played = []
       @player_score = 0
+      @tiles = []
     end
 
     def play_word(word)
```

Comment:
You could simplify this `won?` method to use a single-line conditional like 
`return true if @player_score > 100`